standards-catalogue/source/layouts/standards_table.erb

```diff
@@ -1,36 +1,52 @@
 <% wrap_layout :layout do %>
 
   <div style = "overflow-x:auto;">
-    <% if !current_page.data.title.nil? %>
-      <h1><%= current_page.data.title %></h1>
-    <% else %>
-      <h1><%= current_page.data.name %></h1>
-    <% end %>
+    <h1><%= current_page.data.title %></h1>
     <table style="width:100%">
-      <% current_page.data.each do |key, value| %>
-        <tr>
-          <td nowrap><%= key.titleize.capitalize %>: </td>
-          <% if key == 'link' %>
-            <td nowrap><strong><a href='<%= value %>'><%= value %></a></strong> </td>
-          <% elsif key == 'organisation' %>
-            <td nowrap><strong><%= display_organisation(value) %></strong> </td>
-          <% elsif key == 'status' %>
-            <td nowrap><strong><%= display_status(:standard, value) %></strong> </td>
-          <% elsif key == 'licence_id' %>
-            <%= display_licence(value) %>
-          <% elsif key == 'tags' %>
-            <td nowrap>
-              <ul>
-                <% current_page.data.tags.each do |tag| %>
-                  <li><strong><%= tag %></strong></li>
-                <% end %>
-              </ul>
-            </td>
-          <% else %>
-            <td nowrap><strong><%= value %></strong> </td>
-          <% end %>
-        </tr>
-      <% end %>
+      <tr>
+        <td>Name</td>
+        <td>
+          <strong><%=link_to current_page.data.name, current_page.data.specification_url %></strong>
```

Comment:
Is the difference between `name` and `title` important?

Reply:
no that was a mistake, title shouldn't be in there any more - have changed it to name